domains/hatkidchan.json

```diff
@@ -1,13 +1,13 @@
 {
-  "description": "Static personal page of hatkidchan (hkc) (thx Mia)",
+  "description": "kc personal website",
   "owner": {
     "username": "hatkidchan",
     "email": "hatkidchan@gmail.com",
     "twitter": "hatkidchan",
     "discord": "hkc#0517"
   },
   "record": {
-    "A": ["216.128.176.102"],
+    "A": ["5.181.108.112"],
```

Comment:
I can't reach this IP

Reply:
@hatkidchan I need to know what the IP is for.

Reply:
It is IP of my server. No idea why you can't reach it, but it works for everyone else

![image](https://user-images.githubusercontent.com/35804000/210774139-20e38f57-2db2-4faa-92a9-f4a1c9851ddd.png)

TXT record is *not* used for verification

Reply:
> It's for a server. It's not a webserver

It *is* a webserver and I already configured it a while ago
![image](https://user-images.githubusercontent.com/35804000/210774584-0430473b-2f2e-4348-8808-0982d6aa6981.png)

Reply:
Ok. But what is the TXT for?

Reply:
> Ok. But what is the TXT for?

as already said

> may be small ARG, or may be not. left it for future use

According to [docs](https://www.is-a.dev/docs/domain_structure/) I can put anything I want here, so it's just base64-encoded url

Reply:
Yes it is a Base64 encoded URL

Reply:
> @WilliamDavidHarrison @TerraPlayz i see perfectly **NOTHING** wrong with A records as it can be used for various purposes.

I know MaskDuck i did know that as i have previously worked with them as i am a OS developer and i have used A ip's before